test/src/gtest_hpx_main.cpp

```diff
@@ -40,7 +40,7 @@
 #include <cstdio>
 
 #include <gtest/gtest.h>
-#include <hpx/hpx.hpp>
+#include <hpx/hpx_init.hpp>
```

Comment:
There's an alternative name for this now: `hpx/init.hpp`. What you have there is not deprecated right now, but insider information tells me that we most likely will deprecate `hpx/hpx_init.hpp` (the `hpx_` in `hpx_init` is simply redundant).